### PR TITLE
Makes changes to align Liftoff with new LC101 content

### PR DESCRIPTION
**Pull request by chrisbay**

### CODE REVIEW

**Comment by chrisbay** [comment]
[Address Lifftoff changes needed based on LC101 rework](https://trello.com/c/xVFkfDOm/4-address-lifftoff-changes-needed-based-on-lc101-rework)

**Comment by chrisbay** [comment]
I did my best to find old references, but can't be sure I caught them all. We may want to talk to Impact about asking mentors to help us find any outdated copy or links.

File: src/modules/career/live-coding.rst [hunk 1]
```diff
@@ -75,32 +75,32 @@ Write Syntax
...
-You’ll notice the interviewee is talking through their actions as they
+You'll notice the interviewee is talking through their actions as they
 write code down. This shows the interviewee has the ability to
 communicate effectively with another developer.
 
-As you write syntax, you may find that you still don’t have all of the
+As you write syntax, you may find that you still don't have all of the
 requirements. In this video, the interviewee did not know if the letter
-‘y’ was considered a vowel. To overcome this confusion, they simply asked
+‘y' was considered a vowel. To overcome this confusion, they simply asked
```
Comment (CarlyLanglois) [inline src/modules/career/live-coding.rst]: looks like there's also a smart quote before the y?

**Review by CarlyLanglois — APPROVED** [review]
I didnt re-read the whole site but just checked the changes you made here. looks fine

**Pull request merged**